### PR TITLE
[core] Fix pulling incorrect settings for zmq/map

### PR DESCRIPTION
**Pull request by WinterSolstice8**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Changes some settings calls pulling port wrong for ZMQ

## Steps to test these changes

Login with 2 accounts, use /l /p /t and have them work

### CODE REVIEW

**Review by TeoTwawki — APPROVED** [review]
LGTM

File: src/login/message_server.cpp [hunk 1]
```diff
@@ -251,7 +251,7 @@ void message_server_init()
 
     auto server = fmt::format("tcp://{}:{}",
         settings::get<std::string>("network.ZMQ_IP"),
-        settings::get<std::string>("network.ZMQ_PORT"));
+        settings::get<uint16>("network.ZMQ_PORT"));
```
Comment (zach2good) [inline src/login/message_server.cpp]: This part shouldn't be needed, the get function should do a nice conversion to string if asked

File: src/map/message.cpp [hunk 2]
```diff
@@ -627,7 +627,7 @@ namespace message
 
     void init()
     {
-        init(settings::get<std::string>("network.ZMQ_IP").c_str(), settings::get<uint16>("network.ZMQ_IP"));
+        init(settings::get<std::string>("network.ZMQ_IP").c_str(), settings::get<uint16>("network.ZMQ_PORT"));
```
Comment (zach2good) [inline src/map/message.cpp:630]: Whoops!

**Review by zach2good — APPROVED** [review]

File: src/login/message_server.cpp (hunk 1 above)
Comment (WinterSolstice8) [reply src/login/message_server.cpp]: amended

**Review by zach2good — APPROVED** [review]

**Pull request merged**